hibernate-core/src/main/java/org/hibernate/engine/spi/SessionFactoryDelegatingImpl.java

```diff
@@ -73,11 +75,16 @@ public SessionFactoryDelegatingImpl(SessionFactoryImplementor delegate) {
 		this.delegate = delegate;
 	}
 
-	protected SessionFactoryImplementor getDelegate() {
+	protected SessionFactoryImplementor delegate() {
```

Comment:
This is not a compatibility break as this method was introduced during the 5.2.11 release cycle. I renamed it for consistency after a discussion with Steve.